packages/react-components/react-card/src/components/Card/Card.types.ts

```diff
@@ -124,7 +124,8 @@ export type CardProps = ComponentProps<CardSlots> & {
   /**
    * Callback to be called when the selected state value changes.
    */
-  onSelectionChange?: (event: CardOnSelectionChangeEvent, data: CardOnSelectData) => void;
+  // eslint-disable-next-line @fluentui/consistent-callback-type
+  onSelectionChange?: (event: CardOnSelectionChangeEvent, data: CardOnSelectData) => void; // callback should be typed with EventHandler, but we can't break existing callbacks
```

Comment:
the recommended pattern is to provide inline reasoning when disabling particular rule

```suggestion
  // eslint-disable-next-line @fluentui/consistent-callback-type - callback should be typed with EventHandler, but we can't break existing callbacks
  onSelectionChange?: (event: CardOnSelectionChangeEvent, data: CardOnSelectData) => void;
```

Reply:
Done :)